src/ringmesh/basic/logger.cpp

```diff
@@ -42,6 +42,6 @@
 
 namespace RINGMesh
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty namespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS3261)